webapp/portlet/src/main/webapp/idm-users-management/components/UsersManagementList.vue

```diff
@@ -80,6 +95,15 @@ export default {
     initialized: false,
     loading: true,
     error: null,
+    fullDateFormat: {
+      day: 'numeric',
+      month: 'short',
+      year: 'numeric',
+    },
+    dateTimeFormat: {
+      hour: '2-digit',
+      minute: '2-digit',
+    }
```

Comment:
You can combine both

```suggestion
    fullDateFormat: {
      day: 'numeric',
      month: 'short',
      year: 'numeric',
      hour: '2-digit',
      minute: '2-digit',
    }
```

Reply:
Updated! thanks @boubaker